cfgov/templates/wagtailadmin/css/fig-admin.css

```diff
@@ -0,0 +1,15 @@
+.fig-block {
+    padding: 10px;
+}
+
+.fig-section {
+    background-color:#f0f8eb;
+}
+
+.fig-sub-section {
```

Comment:
I like this (and I hope designers do too) -- it seems like it will help visually parse the layers of blocks on the page. Is there a way to make the colors extend all the way to the edges of the block outlines?